### ISSUE TITLE
[CLI] TensorBoard integration

### ISSUE THREAD

**Issue by sachinshelke** [cloud-annotations/training#103]
Just a quick query
Can it be possible see the eval report while training progress? (tensorboard progress status)

**Comment by bourdakos1** [comment]
If you run cacli logs it will print out the loss and other logs, but I haven’t had the chance to try hooking it up to tensorboard. It should save the usual tensorboard logs into your object storage bucket, if you find a way to pipe that into tensorboard let me know and I can write up some documentation to make it easier for others :)

**Comment by bourdakos1** [comment]
This talks about s3, but IBM’s cloud object storage using the same api structure so you should be able to follow this: https://github.com/tensorflow/examples/blob/master/community/en/docs/deploy/s3.md

**Comment by bourdakos1** [comment]
So set these environment variables:
```
AWS_ACCESS_KEY_ID=your-ibm-access-key
AWS_SECRET_ACCESS_KEY=your-ibm-secret
S3_ENDPOINT=https://s3.us.cloud-object-storage.appdomain.cloud
```

And run tensorboard like this:
```
tensorboard --logdir s3://bucketname/path/to/model/
```

**Comment by sachinshelke** [comment]
Thanks I will try.

**Comment by sachinshelke** [comment]
aws cli has some issue. But will figure it out and keep here posted.

**Comment by bourdakos1** [comment]
We can leave this open until I have some documentation for this

**Comment by sachinshelke** [comment]
Awesome. Will check and revert back once new training started in couple of days.

**Comment by bourdakos1** [comment]
```
cacli tensorboard <model-id>
```
added in cacli v1.1.11

**Comment by sachinshelke** [comment]
Unfortunately didn't work for me with `cacli tensorboard <model-id>`

<img width="726" alt="Screenshot 2019-11-19 at 4 57 54 PM" src="https://user-images.githubusercontent.com/869601/69142884-c2cd9000-0aed-11ea-8ed4-f2368c2431df.png">
<img width="912" alt="Screenshot 2019-11-19 at 4 58 47 PM" src="https://user-images.githubusercontent.com/869601/69142994-fd372d00-0aed-11ea-80ea-068129ece905.png">

**Comment by bourdakos1** [comment]
It can take a while for the data to show up, try refreshing a couple times. Also are there any error messages in terminal?

**Comment by sachinshelke** [comment]
No, I'm not getting any error message.
I'm training model for classification. May be it will not work for classification.

**Comment by bourdakos1** [comment]
ah, yea forgot about classification...

**Comment by sachinshelke** [comment]
It works with localization.

**Comment by bourdakos1** [comment]
I'm going to close this and open an issue for adding classification support
